webpack.preact.config.js

```diff
@@ -0,0 +1,154 @@
+const defaultConfig = require("./node_modules/@wordpress/scripts/config/webpack.config");    // Require default Webpack config
```

Comment:
Duplicating the entire webpack config might not be the best way to handle this :thinking: I guess each of the things webpack is building should use either WordPress's React or Preact, depending on whether the script will be served on the front end or the back end.

Reply:
Definitely possible, but hard to find relevant documentation or examples. Closest I could find is [this plugin that allows to map depending on file name](https://github.com/heiseshandian/dynamic-alias-resolve-plugin#readme). Though it seems to expect the function to return a absolute path, while we want to use another alias.